examples/water_gromacs/water.py

```diff
@@ -36,6 +36,86 @@
 # This example reads in a gromacs water system (SPC/Fw) treated with reaction field. See the corresponding gromacs grompp.mdp paramter file.
 # Output of gromacs energies and esp energies should be the same
 
+def computeRDF(system, rdf_pair, rdf_dr, num_particles, Lx, Ly, Lz, types, atomtypes):
+    list_t = []
+    M_PI = 3.1415926535897932384626433832795029
+    type_end = []
+    type_end.append(0)
+    for k in range(len(atomtypes)):
+        for pid in range(num_particles):
+            if types[pid]==k:
+                list_t.append(pid+1)
+        type_end.append(len(list_t))
+    #print(list_t)
+    #print(type_end)
+    rdf_maxr = min(Lx,Ly,Lz)
+    rdf_bin = int(0.5*rdf_maxr/rdf_dr)
+    
+    rdf_array = []
+    for k in range(len(rdf_pair)):
+        arr_tmp=[0]*rdf_bin
+        rdf_array.append(arr_tmp)
+        del arr_tmp
+
+    pos = espressopp.analysis.Configurations(system)
+    # pos.capacity=1
```

Comment:
Remove?

Reply:
I would uncomment this. Users may keep collecting coordinates which lead to memory leakage for long runs.